### PR TITLE
fix(pfd): tweak TCAS fail position

### PR DESCRIPTION
**Pull request by beheh**
## Summary of Changes

Fix the TCAS fail indication on the PFD by making the letters slightly smaller and moving them closer to each other, like in the attached reference.

## Screenshots (if necessary)

| Before | After |
| ----------- | ----------- |
|  TCAS before | TCAS after |

## References

TCAS fail

## Additional context

Discord username (if different from GitHub):

## Testing instructions

Visual change only. Fail the TCAS e.g. by failing both RAs, observe the indication matching the reference.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by 2hwk — APPROVED** [review]

**Comment by lukecologne** [comment]
While we're at it, we should add the TCAS flag to the PFD svg so that it stays consistent

File: src/instruments/src/PFD/AltitudeIndicator.tsx [hunk 1]
```diff
@@ -209,10 +209,10 @@ export class AltitudeIndicatorOfftape extends DisplayComponent<AltitudeIndicator
...
                 </g>
                 <g ref={this.tcasFailed} style="display: none">
-                    <text class="Blink9Seconds FontLargest Amber EndAlign" x="141.5" y="96">T</text>
-                    <text class="Blink9Seconds FontLargest Amber EndAlign" x="141.5" y="104">C</text>
-                    <text class="Blink9Seconds FontLargest Amber EndAlign" x="141.5" y="112">A</text>
-                    <text class="Blink9Seconds FontLargest Amber EndAlign" x="141.5" y="120">S</text>
+                    <text class="Blink9Seconds FontLarge Amber EndAlign" x="141.5" y="100">T</text>
+                    <text class="Blink9Seconds FontLarge Amber EndAlign" x="141.5" y="105.5">C</text>
+                    <text class="Blink9Seconds FontLarge Amber EndAlign" x="141.5" y="110.75">A</text>
+                    <text class="Blink9Seconds FontLarge Amber EndAlign" x="141.5" y="116">S</text>
```
Comment (lukecologne) [inline src/instruments/src/PFD/AltitudeIndicator.tsx:215]:
```suggestion
                    <text class="Blink9Seconds FontMedium Amber EndAlign" x="141.5" y="100">T</text>
                    <text class="Blink9Seconds FontMedium Amber EndAlign" x="141.5" y="105">C</text>
                    <text class="Blink9Seconds FontMedium Amber EndAlign" x="141.5" y="110">A</text>
                    <text class="Blink9Seconds FontMedium Amber EndAlign" x="141.5" y="115">S</text>
```
This fits a little better IMO: 
Unbenannt

**Comment by tracernz** [comment]
@beheh I took the liberty of adding the text to the PFD SVG before this gets merged in accordance with Luke's request. Hope that's helpful... if not feel free to discard the commit.

**Comment by beheh** [comment]
@tracernz Not at all, thanks for your help! I simply didn't get around to it yet.

**Comment by lukecologne** [comment]
Alright, I think this should be good to go then. Don't think we really need any testing for this.

**Review by lukecologne — APPROVED** [review]

**Pull request merged**